packages/smart-contracts/src/contracts/ChainlinkConversionPath.sol

```diff
@@ -228,4 +230,12 @@ contract ChainlinkConversionPath is WhitelistAdminRole {
     }
     return (size > 0);
   }
+
+  /**
+   * @notice Update the native token hash
+   * @param _nativeTokenHash hash of the native token represented as an eth address
+   */
+  function updateNativeTokenHash(address _nativeTokenHash) external onlyWhitelistAdmin {
```

Comment:
We need to be able to administrate the `_nativeTokenHash`. Since it is different on all chains, we initiate the contract with a null value; then it is updated during the setup. If we don't proceed this way, the contract won't have the same address on all network.